### DISCUSSION TITLE
您好，我想问一下关于这个是什么问题呢？

### DISCUSSION THREAD

**Discussion by autisn330** [lsky-org/lsky-pro#393] [问答]
这边显示上传成功
![image](https://user-images.githubusercontent.com/92194759/164355211-5515a406-285a-4ceb-a1cc-4c8aeda8c093.png)

我在七牛云也看到了该文件
![image](https://user-images.githubusercontent.com/92194759/164355265-11bf5efd-cf2a-472c-9b55-734fcca141cb.png)

可我复制该链接出来时，图片显示404
![image](https://user-images.githubusercontent.com/92194759/164355332-3e086310-148d-4a53-b3e3-f1f0d388b0e2.png)

**Comment by 0xxb** [comment]
你的域名解析错了吧？七牛云的储存架构是 openresty，而不是单独的 nginx

**Reply by autisn330** [reply]
我用的是腾讯云的域名解析
![image](https://user-images.githubusercontent.com/92194759/164356369-059bf9d3-ecc2-4c9c-86cf-819044129bf0.png)
![image](https://user-images.githubusercontent.com/92194759/164356508-bfe78ec0-4e77-408d-b64d-1ba52bcd2d94.png)
是这样的嘛
